packages/itwinui-css/src/progress-indicator/linear.scss

```diff
@@ -11,7 +11,7 @@
     width: 100%;
     height: $height;
     overflow: hidden;
-    background-color: var(--iui-color-background-border);
+    background-color: var(--iui-color-background-5);
```

Comment:
Changed so the filled track is more visible in high contrast mode.

**Old, using background border:**
![Screen Shot 2022-09-15 at 2 45 32 PM](https://user-images.githubusercontent.com/849817/190485055-5773d71d-cdc2-4475-8d6c-bb5ecf8da86e.png)
![Screen Shot 2022-09-15 at 2 45 39 PM](https://user-images.githubusercontent.com/849817/190485057-e0248c1b-fa16-4c28-8ec7-db70f29028fd.png)

**New, using background 5:**
![Screen Shot 2022-09-15 at 2 45 06 PM](https://user-images.githubusercontent.com/849817/190485022-c2c034e3-3f34-4298-94cf-e143826a1968.png)
![Screen Shot 2022-09-15 at 2 45 13 PM](https://user-images.githubusercontent.com/849817/190485027-8d2d9f0a-b9b3-4d87-9062-9dc8ec272c0b.png)